homework/02LessonTwo.js

```diff
@@ -126,7 +177,8 @@ function whichSpecies(character) {
 //
 // In your own words, explain what conditionals do.
 //
-// Your Answer Goes Here:
+// A conditional is a statement that relies on the information given to it in an "if" "else" form. It does
+//different things under different conditions.
```

Comment:
You're on the right track!

![yay](http://a.fod4.com/misc/anchorman-teamjump.gif)